Mathlib/CategoryTheory/Sites/EqualizerSheafCondition.lean

```diff
@@ -179,12 +181,14 @@ variable [R.hasPullbacks]
 The rightmost object of the fork diagram of https://stacks.math.columbia.edu/tag/00VM, which
 contains the data used to check a family of elements for a presieve is compatible.
 -/
-@[simp] def SecondObj : Type max v u :=
+@[simp, stacks 00VL "This is the rightmost object of the fork diagram there."]
```

Comment:
```suggestion
@[simp, stacks 00VM "This is the rightmost object of the fork diagram there."]
```
I checked: `00VL` is the section, but the definition is `00VM`.

Reply:
Yes, `00VL` has quite a few fork diagrams while `00VM` has exactly one, so `00VM` is the useful tag.

Reply:
Yeah, just a typo